torch/lib/c10d/reducer.cpp

```diff
@@ -1035,6 +1051,18 @@ void Reducer::copy_bucket_to_grad(
   });
 }
 
+std::vector<std::string> Reducer::getUnmarkedParamsForIteration() {
+  TORCH_INTERNAL_ASSERT(ddp_debug_level_ != DistributedDebugLevel::OFF);
+  std::vector<std::string> unMarkedParamNames;
+  for (const auto& it : param_names_) {
```

Comment:
in default mode, return indices. in debug mode, return indices plus names.

Reply:
Will add this logic in next update